src/main/java/com/getindata/kafka/connect/iceberg/sink/IcebergChangeEvent.java

```diff
@@ -419,9 +428,9 @@ private List<Types.NestedField> icebergSchema(JsonNode eventSchema, String schem
         }
       }
 
-      if (addSourceTsField) {
+      if (addPartitionField) {
         columnId++;
-        schemaColumns.add(Types.NestedField.optional(columnId, "__source_ts",
+        schemaColumns.add(Types.NestedField.optional(columnId, partitionColumn,
```

Comment:
Hmm, so choosing a different partition column means no longer capturing `__source_ts`?

Reply:
It is the default name.  The upstream change allows that column name to be changed.